### DISCUSSION TITLE
Welcome to OpenMIDIStomper Discussions!

### DISCUSSION THREAD

**Discussion by silveirago** [silveirago/OpenMIDIStomper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bellows22** [comment]
Hi,
LooseBruce here. I use a Roland V-Accordion as my midi controller. I have been using iPad with AUM for home music production for several years. I decided recently to use some hardware as opposed to mobile apps. I bought a MicroDexed Touch module and am learning about it. Now I’m assembling an open source midi stomper. Having build Nerd Musician projects in the past I feel pretty confident. Just thought I would lay the groundwork for a question that is bound to come up before I am done. Thanks
